### ISSUE TITLE
Autologin Feature

### ISSUE THREAD

**Issue by faspie** [grommunio/grommunio-desktop#2]
I would like to suggest a feature for an automatic login on the startup of the deskop app. E. g. Windows credentials could be used, or a "save password" feature.

**Comment by Kaelber** [comment]
Same from my side.

Default settings in the login are desirable in the DeskAPP. This is already possible in the WebAPP via a browser. Could a "remember password" function be set up? Or something similar that makes it easier for the user to log in?

**Comment by Kaelber** [comment]
No comments?

**Comment by jengelh** [comment]
Community sites are monitored and/or acted upon on a best-effort basis (like in any open source project). For now, this feature won't receive priority.

**Comment by faspie** [comment]
Why does this feature not receive priority? User acceptance raises and falls with a convenient use, and logging in twice is not very convenient.  Even in kopano-desktop this feature is implemented since years.

**Comment by Kaelber** [comment]
I agree with that. From my point of view the DeskAPP is basically more important than the WebAPP, as it can be used in much the same way as Outlook. I also have to deal with several accounts with my customers, so Outlook is usually busy and the DeskAPP is therefore a great help.
